lib/browser/api/auto-updater/squirrel-update-win.ts

```diff
@@ -15,89 +15,67 @@ const isSameArgs = (args: string[]) => args.length === spawnedArgs.length && arg
 
 // Spawn a command and invoke the callback when it completes with an error
 // and the output from standard out.
-const spawnUpdate = function (args: string[], detached: boolean, callback: Function) {
-  let error: Error, errorEmitted: boolean, stderr: string, stdout: string;
-
-  try {
+const spawnUpdate = async function (args: string[], detached: boolean): Promise<string> {
```

Comment:
a bit of a nitpick, but if we're here anyway we might avoid this boolean trap and refactor this to `async function (args: string[], options: { detached: boolean })`